src/test/java/org/mozilla/taskcluster/APITest.java

```diff
@@ -138,4 +141,59 @@ public void testDefineTask() {
 
         System.out.println("=> Task https://queue.taskcluster.net/v1/task/" + taskId + " created successfully");
     }
+
+    /*
+     * This just checks if the TemporaryCredentials works
+     */
+    @Test
+    public void createTemporaryCredentials(){
+      try{
+        String[] tempScopes = new String[2];
+        Date start = new Date();
+        Date expiry = new Date(start.getTime() + 2*24*60*60*1000);
+        tempScopes[0] = "scopeA";
+        tempScopes[1] = "scopeB";
+        String temp = TemporaryCredentials.createCredentials(
+        "clientId","tokenABCDEFGH", tempScopes, start, expiry
+        );
+        Gson gson = new Gson();
+        System.out.println("=> unnamed credentials");
+        Credentials cred = gson.fromJson(temp,Credentials.class);
+        System.out.println("accessToken: "+ cred.accessToken);
+        System.out.println("clientId: "+ cred.clientId);
+        System.out.println("certificate.signature: "+cred.certificate.signature);
+        System.out.println("certificate.seed: "+cred.certificate.seed);
+        System.out.println("certificate.start: "+cred.certificate.start);
+        System.out.println("certificate.expiry: "+cred.certificate.expiry);
+        System.out.print("scopes: ");
+        for(String scope: cred.certificate.scopes){
+          System.out.print(scope + " ");
+        }
+
+        temp = TemporaryCredentials.createCredentials(
+        "clientId","issuer","tokenABCDEFGH", tempScopes, start, expiry
+        );
+
+        System.out.println("=> named credentials");
+        cred = gson.fromJson(temp,Credentials.class);
+        System.out.println("accessToken: "+ cred.accessToken);
+        System.out.println("clientId: "+ cred.clientId);
+        System.out.println("certificate.clientId: "+cred.certificate.clientId);
+        System.out.println("certificate.issuer: "+cred.certificate.issuer);
+        System.out.println("certificate.signature: "+cred.certificate.signature);
+        System.out.println("certificate.seed: "+cred.certificate.seed);
+        System.out.println("certificate.start: "+cred.certificate.start);
+        System.out.println("certificate.expiry: "+cred.certificate.expiry);
+        System.out.print("scopes: ");
+        for(String scope: cred.certificate.scopes){
+          System.out.print(scope + " ");
+        }
+        //System.out.println("=>unnamed credentials "+temp);
+      }catch(Exception e){
+        e.printStackTrace();
+        Assert.fail("Exception thrown");
+      }
+      System.out.println("\n=> TemporaryCredentials\n");
+    }
```

Comment:
I'm not sure how much value this test has, maybe we could test against the Auth test endpoints?